syntax = "proto3";

package YGGFY;

service MyMegaService {
  rpc CalculateValue(Request) returns (Response);
}

message Request {
  int32 value1 = 1;
  int32 value2 = 2;
}

message Response {
  int32 value = 1;
}